recipe/meta.yaml

```
@@ -208,14 +209,15 @@ outputs:
       build:
         - {{ compiler('m2w64_c') }}    # [win]
         - {{ compiler('m2w64_cxx') }}  # [win]
+        - {{ stdlib('m2w64_c') }}      # [win]
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ stdlib('c') }}
```

This probably needs to be restricted to unix.
```suggestion
        - {{ stdlib('c') }}            # [unix]
```

---

Sure let's try :)